src/main/java/org/folio/search/service/browse/CallNumberBrowseService.java

```diff
@@ -84,6 +80,29 @@ protected String getValueForBrowsing(CallNumberBrowseItem browseItem) {
     return browseItem.getShelfKey();
   }
 
+  private BrowseResult<CallNumberBrowseItem> getPrecedingResult(BrowseRequest request,
+                                                                BrowseContext context,
+                                                                SearchSourceBuilder precedingQuery,
+                                                                MultiSearchResponse multiSearchResponse) {
+    var responses = multiSearchResponse.getResponses();
+    var precedingResult = callNumberBrowseResultConverter.convert(responses[0].getResponse(), context, false);
+    var succeedingPrecedingResult = callNumberBrowseResultConverter.convert(responses[1].getResponse(), context, false);
```

Comment:
Maybe we should name it "backwardPrecedingResult"

Reply:
Changed.